src/Services/Passenger/src/Passenger/Passengers/Models/ValueObjects/Age.cs

```diff
@@ -0,0 +1,18 @@
+namespace Passenger.Passengers.Models.ValueObjects;
+using System;
+
+public record Age : GenericValueObject<int>
```

Comment:
Please remove generic approach for this value object.

Reply:
done